tests/src/test/java/alluxio/client/FileSystemIntegrationTest.java

```diff
@@ -79,7 +79,7 @@ public void createFileWithFileAlreadyExistsException() throws Exception {
     mFileSystem.createFile(uri, mWriteBoth).close();
     Assert.assertNotNull(mFileSystem.getStatus(uri));
     try {
-      mFileSystem.createFile(uri, mWriteBoth);
+      mFileSystem.createFile(uri, mWriteBoth).close();
```

Comment:
The RetryHandlingBlockWorkerClientTestUtil#reset is to ensure this. For all the tests that use the local cluster, if there is any resource not clean up, the test fails.